build-system/conformance-config.textproto

```diff
@@ -168,6 +168,22 @@ requirement: {
 }
 
 requirement: {
+  type: BANNED_PROPERTY_READ
```

Comment:
It is a property on `CustomElement` though. We should probably change action-impl's implementation instead, e.g. copy properties into a new `CustomElement` object like we do for `DeferredEvent`.